apollo-router/src/plugins/telemetry/snapshots/apollo_router__plugins__telemetry__tests__it_test_prometheus_metrics.snap

```diff
@@ -1,8 +1,17 @@
 ---
 source: apollo-router/src/plugins/telemetry/mod.rs
-expression: prom_metrics
+expression: prometheus_metrics
```

Comment:
I don't really understand this change, but won't this impact existing consumers of `http_request_duration`?

Reply:
The metric was always called: `apollo_router_http_request_duration_seconds_bucket`. Pretty unfortunate.

In terms of why the snapshot has changed, it's because we no longer specify lots of extra attributes just for this prometheus test. The test is focused just on did prometheus serve up a result and did it look OK.

Extra attributes are still tested in the new tests that use the testing macros, such as: `test_supergraph_metrics_ok`